sdk/azcore/tracing/tracing.go

```diff
@@ -106,13 +127,25 @@ func (t Tracer) SpanFromContext(ctx context.Context) Span {
 	return Span{}
 }
 
+// LinkFromContext returns a link encapsulating the SpanContext of the current context.
+// If the provided context has no Span, an empty Link is returned.
+func (t Tracer) LinkFromContext(ctx context.Context, attrs ...Attribute) Link {
```

Comment:
What happens with `attrs` here, are they added to the returned `SpanContext`? Might be good to document how this arg is used.

Reply:
Added a short description to what attrs are used for